README.adoc

```diff
@@ -29,13 +29,27 @@ buildPlugin()
   execute the steps against in parallel
 * `jenkinsVersions`: (default: `[null]`) - a matrix of Jenkins baseline versions to build/test against in parallel (null means default,
   only available for Maven projects)
+* `findbugs`: (default: `null`) - a map of parameters to run findbugs and/or archive findbugs reports. (only available for Maven projects)
+** `run`: set to `true` to add the `findbugs:findbugs` goal to the maven command.
```

Comment:
Rather, you do not need to `run` it because it is already run by default. Rather we want to archive the results by default. (And allow that plugin to mark the build as `UNSTABLE` rather than `FAILURE`.)